### ISSUE TITLE
Seed is ignored in python package for gpu_hist

### ISSUE THREAD

**Issue by jfpuget** [dmlc/xgboost#6347]
Using xgboost installed via rapids 0.16, with thes eparamaters:

params =  {
  "objective"           : "binary:logitraw",
  "eval_metric"         : "logloss",
    'tree_method'       : 'gpu_hist',
}

then setting params['seed'] to different values before calling xgb.train() always produce the same effect.

**Comment by jfpuget** [comment]
False alarm, sorry.
